test-suite-kotlin-ksp/src/test/kotlin/io/micronaut/docs/server/suspend/SuspendController.kt

```diff
@@ -17,10 +17,8 @@ package io.micronaut.docs.server.suspend
 
 import io.micronaut.http.*
 import io.micronaut.http.annotation.*
-import io.micronaut.http.bind.binders.HttpCoroutineContextFactory
 import io.micronaut.http.context.ServerRequestContext
 import io.micronaut.scheduling.TaskExecutors
-import io.micronaut.tracing.instrument.kotlin.CoroutineTracingDispatcher
```

Comment:
`CouroutineTracingDispatcher` was deleted here https://github.com/micronaut-projects/micronaut-tracing/pull/281/files

No test was using it.